.github/workflows/commitlint.yml

```diff
@@ -9,13 +9,12 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - run: env | sort
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
       - name: Validate the latest commit message with commitlint
         if: github.event_name == 'push'
-        run: echo "${{ github.event.head_commit.message }}" | npx commitlint
+        run: echo "${{ github.event.head_commit.message }}" | npx commitlint -x @commitlint/config-conventional
```

Comment:
why not use npx --package same as the semantic-release.🤔

Reply:
Please refer to this: https://github.com/serious-scaffold/ss-python/pull/562

Now we are useing the container instead, so it is also managed by renovate, too.

Reply:
Got it.😬